designer/client/components/graph/node-modal/editors/expression/SpelTemplateEditor.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {useMemo} from "react"
+import RawEditor, {RawEditorProps} from "./RawEditor"
+import {ExpressionLang} from "./types";
+
+const SpelTemplateEditor = (props: RawEditorProps) => {
+
+  const {expressionObj, ...passProps} = props
+
+  const value = useMemo(() => ({
+    expression: expressionObj.expression,
+    language: ExpressionLang.SpEL,
```

Comment:
why not `ExpressionLang.SpELTemplate` since it is SpelTemplateEditor?

Reply:
It is tricky, but this language is for purpose of syntax highlighting. We don't have two themes for that - only one, spel. I'll add a comment there.